routes/user.js

```diff
@@ -2,100 +2,122 @@
 
 const AppConstants = require("../app-constants");
 
-const crypto = require("crypto");
 const express = require("express");
 const router = express.Router();
 
 const EmailUtils = require("../email-utils");
 const Subscribers = require("../subscribers");
 
-// We send verification emails to addresses that want to subscribe.
-// These addresses are temporarily stored here, mapped to the unique
-// string token that is used for verification.
-const gUnverifiedEmails = new Map();
+const ResponseCodes = Object.freeze({
```

Comment:
Is there a doc or some other link that describes where these error codes came from? Can we include it in a comment here?

Reply:
Yeah, I'm going to update api.md